### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by dyladan**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@opentelemetry/instrumentation-nestjs-core: 0.28.1</summary>

### Bug Fixes

* **opentelemetry-instrumentation-nestjs-core:** copy metadata to wrapped handler ([#796](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/796)) ([2c4a834](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/2c4a83421c979f65fd464d9599882de9a65bbc74))
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by dyladan** [comment]
@open-telemetry/javascript-approvers I need an approval to merge this

**Review by Flarna — APPROVED** [review]

**Comment by dyladan** [comment]
:robot: Release for @opentelemetry/instrumentation-nestjs-core is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-nestjs-core-v0.28.1 :sunflower:

**Pull request merged**
